docs/backend/upgrading/upgrade-to-volto.md

```diff
@@ -10,54 +10,54 @@ html_meta:
 
 ```{todo}
 Is this "upgrading" or "migrating"? It seems to be a migration more than an upgrade. Please clarify.
+
+Technically it is a migration since the version of Plone does not get changed and it is no upgrade-step.
+It could be called a upgrade though because Volto is the new default frontend.
+Not 100% sure...
 ```
 
+
 # Upgrading to Volto
 
-```{todo}
-I think there needs to be an introduction to this chapter to give it better context. Here are a few questions I have when I jump to this document from who knows where.
+Plone 6 comes with a new default frontend, called Volto.
+Volto is written in React and uses `plone.restapi` to communicate with the backend.
+When creating a new site in Plone 6 you have the option to choose between creating a Volto site and a Classic Plone site.
```

Comment:
I actually is "Create a new Plone site" and "Create Classic Plone site":

<img width="571" alt="Bildschirmfoto 2022-05-17 um 19 06 53" src="https://user-images.githubusercontent.com/453208/168870925-5f93b9af-87c4-424a-868a-b2f871f9aa77.png">

I'll update accordingly

Reply:
Where would be a good place to discuss this UI? I think it could be improved. I approach this with fresh eyes, and it is... confusing.

For this PR, aligning with the UI is the way forward.

Reply:
Found it, and not easily.

https://github.com/plone/Products.CMFPlone/commit/cd57b6e82fca76a35caac018a18211e7a7dddc39